text/0014-facet-number.md

```diff
@@ -0,0 +1,85 @@
+- Title:
+- Start Date:
+- Specification PR:
+- MeiliSearch Issue: 
+
+# Facet Numbers / Facet Range
+
+## 1. Feature Description and Interaction
+
+### Summary
+
+Handle additional operation on Number typed Facets like:
+- higher to
+- lower to
+- equal to
+- higher or equal to
+- lower or equal to
+
+Handle additional `facetStats` on Number typed Facets like:
+- min
+- max
+- sum (optional?)
+- average (optional?)
+
+Provide the user a way to configure it.
+
+### Motivation
+
+As an e-commerce platform, I want to provide a **Price Slider** to be able to filter products on a price range;
+
+As an e-commerce platform, I want to provide a **Customer Reviews Rank** (`★★★☆☆`) to be able to filter products with a minimum rank;
+
+### Additional Materials
+### Explanation
+
+To create a **Price Slider** we would need a least:
+- higher or equal to
+- lower or equal to
+to filter on a range of price and:
+- min
+- max
+
+To provide the minimum and maximum price, the End User can choose.
+
+To create a **Customer Reviews Rank** we would need a least:
+- higher or equal to
+
+To filter products with a minimum rank.
+
+### Impact on Documentation
+
+Changes on the `search` route request adding new operators:
+- higher to
+- lower to
+- equal to
+- higher or equal to
+- lower or equal to
+
+
+Changes on the `search` route response adding new `facetStats`:
+- min
+- max
+- sum
+- average
+
+Changes in `settings` route changing `AttributeForFaceting` to handle Number typed Facets.
+
+## 2. Technical Specifications
+
+### Architecture
+### Implementation Details
```

Comment:
@Kerollmops can you add some details of your implementation?

Reply:
Tell me if you want me to change something, https://github.com/meilisearch/specifications/pull/14#issuecomment-735835847.

Reply:
Thanks, I added it in the spec.